pkg/linter/linterExecute.go

```diff
@@ -17,9 +17,25 @@ import (
 const issues = "issues:"
 const delimBetweenLintsChars = 200
 
+func GetLintersNames() map[string]bool {
+	return map[string]bool{blockedTCPResponse: true, splitRuleSubnetNACLName: true, splitRuleSubnetSGName: true,
+		overlappingSubnetsName: true, ruleNonRelevantCIDRNACLName: true, redundantTablesName: true,
+		ruleNonRelevantCIDRSGName: true}
+}
+
 // LinterExecute executes linters one by one
-// todo: mechanism for disabling/enabling lint checks
-func LinterExecute(configs map[string]*vpcmodel.VPCConfig) (issueFound bool, resString string, err error) {
+func LinterExecute(configs map[string]*vpcmodel.VPCConfig,
+	enableList, disableList []string) (issueFound bool, resString string, err error) {
+	enableLinters := // enabling and disabling linters defaults
+		map[string]bool{blockedTCPResponse: true, splitRuleSubnetNACLName: true, splitRuleSubnetSGName: true,
```

Comment:
Didn't we say `splitRuleSubnetSGName` should be off by default?

Reply:
Yes, we did. Changed default.